fw-child/resources/js/child-functions.js

```diff
@@ -609,47 +609,77 @@ const $ = jQuery;
     
       $('.scroll-card').each(function () {
         const card = this;
-    
+        const isLastCard = $(card).closest('.fw-query-item').is(':last-child');
+
+         // Set pointer-events to none at the beginning for every card
+        $(card).css('pointer-events', 'none');
+        
+        // Define common properties
+        let fromToProps = { 
+          y: '-85%',
+          opacity: 0,
+          scale: 0.75,
+          zIndex: 0
+        };
+        let toProps1 = {
```

Comment:
I would add an empty line between this variable and the preceding. Same with the next variable.